packages/cactus-core-api/src/main/typescript/plugin/consortium/i-plugin-consortium.ts

```diff
@@ -0,0 +1,29 @@
+import { ICactusPlugin } from "../i-cactus-plugin";
+
+/**
+ * Common interface to be implemented by plugins which are implementing the connection to ledgers.
+ */
+export interface IPluginConsortium<
+  GetNodeJwsRequest,
```

Comment:
@jordigiam Generic type parameters having the exact same name as the classes can be a bit confusing in other places of the code where the actual classes will be imported alongside the type parameters being mentioned. Any chance that you could give them some other name in the interest of disambiguation? Could go with just the old school single letter kind of things from java like `T1, T2, T3, T4` but I also think that those are not very expressive so maybe something in-between that still conveys the meaning and yet different from the actual class names like `GetNodeJwsIn, GetNodeJwsOut, GetConsortiumJwsIn, GetConsortiumJwsOut`. These also have the added benefit that the in/out are more generic that can apply to function calls whereas the words `response` and `request` always make (me) think of HTTP only.

Reply:
Sure! Makes sense.